pkg/cmd/server/start/bootstrap_node.go

```diff
@@ -252,35 +173,69 @@ func (o NodeOptions) loadBootstrap(hostnames []string, nodeConfigDir string) err
 
 	// if we had a previous node-config.yaml, continue using it
 	if _, err2 := os.Stat(o.ConfigFile); err2 == nil {
-		if err == nil {
-			glog.V(2).Infof("Unable to load node configuration from the server: %v", err)
+		if err != nil {
+			glog.Warningf("Unable to load node configuration from the server: %v", err)
 		}
+
 		return nil
 	}
 
+	switch {
+	case kerrors.IsNotFound(err):
+		err = nil
+	case kerrors.IsForbidden(err):
+		glog.Warningf("Node is not authorized to access config, continuing without config: %v", err)
+		err = nil
```

Comment:
So you requested to bootstrap, you attempted the lookup, but your setup was bad and you couldn't find the configmap.  This looks like it should fail.